packages/react/hover-card/src/HoverCard.stories.tsx

```diff
@@ -0,0 +1,696 @@
+  return (
+    <HoverCard>
+      <HoverCardAnchor
+        style={{
+          display: 'flex',
+          justifyContent: 'space-between',
+          alignItems: 'center',
+          width: 250,
+          padding: 20,
+          margin: 100,
+          backgroundColor: '#eee',
+        }}
+      >
+        Item
+        <HoverCardTrigger href="/" className={triggerClass}>
+          trigger
+        </HoverCardTrigger>
+      </HoverCardAnchor>
+      <HoverCardContent
+        className={contentClass}
+        side="right"
+        sideOffset={1}
+        align="start"
+        style={{ borderRadius: 0, width: 200, height: 100 }}
+      >
+        content
+      </HoverCardContent>
+    </HoverCard>
+  );
+};
+
+export const NonPortal = () => {
+  return (
+    <HoverCard>
+      <HoverCardTrigger href="/" className={triggerClass}>
+        trigger
+      </HoverCardTrigger>
+      <HoverCardContent portalled={false} className={contentClass} sideOffset={5}>
+        <HoverCardArrow className={arrowClass} width={20} height={10} offset={10} />
+        <CardContentPlaceholder />
+      </HoverCardContent>
+    </HoverCard>
+  );
+};
+
+export const WithSlottedTrigger = () => {
+  return (
+    <HoverCard>
+      <HoverCardTrigger as={Slot}>
+        <button className={triggerClass} onClick={() => console.log('StyledTrigger click')}>
+          trigger
+        </button>
+      </HoverCardTrigger>
+      <HoverCardContent className={contentClass} sideOffset={5}>
+        <HoverCardArrow className={arrowClass} width={20} height={10} offset={10} />
+        <CardContentPlaceholder />
+      </HoverCardContent>
+    </HoverCard>
+  );
+};
+
+export const WithSlottedContent = () => (
+  <HoverCard>
+    <HoverCardTrigger href="/" className={triggerClass}>
+      trigger
+    </HoverCardTrigger>
+    <HoverCardContent as={Slot} sideOffset={5}>
+      <div className={contentClass}>
+        <HoverCardArrow className={arrowClass} width={20} height={10} offset={10} />
+        <CardContentPlaceholder />
+      </div>
+    </HoverCardContent>
+  </HoverCard>
+);
+
+// change order slightly for more pleasing visual
+const SIDES = SIDE_OPTIONS.filter((side) => side !== 'bottom').concat(['bottom']);
+
+export const Chromatic = () => (
+  <div style={{ padding: 200 }}>
+    <h1>Uncontrolled</h1>
+    <h2>Closed</h2>
+    <HoverCard>
+      <HoverCardTrigger className={triggerClass}>trigger</HoverCardTrigger>
+      <HoverCardContent className={contentClass} sideOffset={5}>
+        <HoverCardArrow className={arrowClass} width={20} height={10} />
+        Some content
+      </HoverCardContent>
+    </HoverCard>
+
+    <h2>Open</h2>
+    <HoverCard defaultOpen>
+      <HoverCardTrigger className={triggerClass}>trigger</HoverCardTrigger>
+      <HoverCardContent className={contentClass} sideOffset={5}>
+        <HoverCardArrow className={arrowClass} width={20} height={10} />
+        Some content
+      </HoverCardContent>
+    </HoverCard>
+
```

Comment:
Can you also add the "open with reordered parts" stories in both controlled and uncontrolled like in popover, tooltip, etc? 🙏

Reply:
Yep, sorry, we merged these extra ones yesterday which will be why they weren't included in your copy/paste 🙈

Reply:
S'all good, will take a look!